### DISCUSSION TITLE
Grafana newer version

### DISCUSSION THREAD

**Discussion by Hellraiser8** [teslamate-org/teslamate#3249] [Q&A]
I'm using the grafana as well for other dashboard, e.g. my PV system. Unfortunately some features (like common zero line) are only available in V9 of grafana. How can I update only the grafana part or is there a timeline for grafana to be updated to V9 or even V10?

**Comment by DrMichael** [comment]
Latest version with teslamate is 9.5.3. (Since 3 days :-))

**Reply by Hellraiser8** [reply]
really? Changlog says "Bump Grafana to 8.5.26"

**Reply by Hellraiser8** [reply]
anyway I just did a "docker-compose pull" and "up -d" and it is still showing 8.5.26

**Reply by cwanja** [reply]
Looks like 9.5 commit is waiting for a future release. So, it is coming. Not timeline for release.

**Reply by DrMichael** [reply]
You're on     image: teslamate/grafana:edge? My Grafana is on 9.5.3.

https://github.com/adriankumpf/teslamate/commit/07d683ed5a1a12f4e276ab8b4458413a76ecb1a7

**Reply by Hellraiser8** [reply]
teslamate/grafana:latest
